tests/maintel/test_warmup_hexapod.py

```diff
@@ -0,0 +1,125 @@
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import logging
+
+# import random
+import unittest
+import numpy as np
+
+from lsst.ts import utils
+from lsst.ts import standardscripts
+from lsst.ts.externalscripts.maintel import WarmUpHexapod
+from lsst.ts.idl.enums import Script
+
+np.random.seed(42)
+
+index_gen = utils.index_generator()
+
+logger = logging.getLogger(__name__)
+logger.propagate = True
+
+
+class TestCameraHexapodWarmUp(
+    standardscripts.BaseScriptTestCase, unittest.IsolatedAsyncioTestCase
+):
+    def setUp(self):
+        pass
+
+    async def basic_make_script(self, index):
+        logger.debug("Starting basic_make_script")
+        self.script = WarmUpHexapod(index=index)
+
+        # self.script.checkpoints_activities = [
+        #     (checkpoint, unittest.mock.AsyncMock())
+        #     for checkpoint, _ in self.script.checkpoints_activities
+        # ]
+
+        logger.debug("Finished initializing from basic_make_script")
+        # Return a single element tuple
+        return (self.script,)
+
+    async def test_configure(self):
+        async with self.make_script():
+
+            # Try configure with minimum set of parameters declared
+            # Note that all are scalars and should be converted to arrays
+            hexapod = "camera_hexapod"
+            axis = "z"
+            step_size = 250
+            sleep_time = 2.0
+            max_position = 13000
+
+            await self.configure_script(
+                hexapod=hexapod,
+                axis=axis,
+                step_size=step_size,
+                sleep_time=sleep_time,
+                max_position=max_position,
+            )
+
+            assert self.script.config.hexapod == hexapod
+            assert self.script.config.axis == axis
+            assert self.script.config.step_size == [
+                step_size
+            ]  # int/float are converted to lists
+            assert self.script.config.sleep_time == [
+                sleep_time
+            ]  # int/float are converted to lists
+            assert self.script.config.max_position == max_position
+
+            assert self.script.hexapod_name == hexapod
+            assert self.script.sal_index == 1  # camera_hexapod
```

Comment:
Consider checking agains the enumerations.. This way you don't have to update the test if we change the value.